### PR TITLE
Fix version printing to the crate version

### LINKED ISSUE #74: Fix version reporting from tremor commandline tools
**Problem**

Tremor CLI tool (tremor-script, tremor-query, tremor-tool) report old versions.

**Steps**

Run commands like `tremor-script --version`.

**Possible Solution(s)**

Version number is hardcoded in places like:
https://github.com/wayfair-tremor/tremor-runtime/blob/v0.7.3/tremor-script/src/main.rs#L70

Can use `CARGO_PKG_VERSION` environment variable to align the version there with that defined in Cargo.toml. Example:
https://github.com/wayfair-tremor/tremor-language-server/blob/v0.7.3/src/main.rs#L27

### PR DESCRIPTION
**Pull request by Licenser**
closes #74

### CODE REVIEW

File: tremor-tool/src/cli.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: tremor-cli
+name: tremor-tool
 version: "1.0"
```
Comment (anupdhml) [inline tremor-tool/src/cli.yaml]: does the version here change anything for the version determination?

File: tremor-query/src/main.rs [hunk 2]
```diff
@@ -50,7 +50,7 @@ pub fn nanotime() -> u64 {
 #[allow(clippy::too_many_lines)]
 fn main() -> Result<()> {
     let matches = App::new("tremor-query")
-        .version("0.6.0")
+        .version(option_env!("CARGO_PKG_VERSION").unwrap_or(""))
```
Comment (anupdhml) [inline tremor-query/src/main.rs]: nice! didn't know about option_env

File: tremor-tool/src/cli.yaml (hunk 1 above)
Comment (Licenser) [reply tremor-tool/src/cli.yaml]: nope it has no function, good call we might as well kill it

**Comment by Licenser** [comment]
> while we are here, should we also add a `version` flag to tremor-server?

we a;readu got that:
```
     Running `target/debug/tremor-server --version`
tremor version: 0.7.3
rd_kafka version: 0x000000ff, 1.3.0
tremor-runtime 0.7.3
```

**Pull request merged**